1-org/README.md

```diff
@@ -103,6 +103,33 @@ To check if it already exists run:
 gcloud scc notifications describe <scc_notification_name> --organization=<org_id>
 ```
 
+**Note:** This module manages contacts for notifications using [Essential Contacts](https://cloud.google.com/resource-manager/docs/managing-notification-contacts) API. This is assigned at the Parent Level (Organization or Folder) you configured to be inherited by all child resources. Billing notifications are assigned to be sent to `group_billing_admins` mandatory group. Legal and Suspension notifications are assigned to `group_org_admins` mandatory group. If you provide all other groups notifications will be configured like the table below:
+
+| Group | Notification Category | Fallback Group |
+|-------|-----------------------|----------------|
+| gcp_network_viewer | Technical | Org Admins |
+| gcp_platform_viewer | Product Updates and Technical | Org Admins |
+| gcp_scc_admin | Product Updates and Security | Org Admins |
+| gcp_security_reviewer | Security and Technical | Org Admins |
+
+**Note:** There is also possible to assign Essential Contacts directly to projects using project-factory [essential_contacts submodule](https://registry.terraform.io/modules/terraform-google-modules/project-factory/google/13.1.0/submodules/essential_contacts). Here is an example:
+
+```hcl
+module "essential_contacts" {
+  source  = "terraform-google-modules/project-factory/google//modules/essential_contacts"
+  version = "~> 13.0"
+
+  project_id = "<project_id>"
+
+  essential_contacts = {
+    "<security group email>"  = ["SECURITY", "TECHNICAL"],
+    "<financial group email>" = ["BILLING"]
+  }
+
+  language_tag = "en"
+}
+```
```

Comment:
nit: lets remove this as the https://registry.terraform.io/modules/terraform-google-modules/project-factory/google/13.1.0/submodules/essential_contacts has example and we can keep the readme smaller.

Reply:
Done